README.md

```diff
@@ -88,12 +88,15 @@ uom = {
    `rational`, `rational32`, `rational64`, `bigrational`, `f32`, `f64` -- Features to enable
    underlying storage types. At least one of these features must be enabled. `f32` and `f64` are
    enabled by default.
+ * `serde` -- Feature to enable support for serialization and deserialization of quantities with
+   the [serde][serde] crate.
```

Comment:
Note that the feature is disabled by default. Also include the `serde` feature in the `toml` block above: `"serde", # Serde support.`